model/project.go

```diff
@@ -736,6 +736,23 @@ func FindProjectFromTask(t *task.Task) (*Project, error) {
 	return p, nil
 }
 
+func FindProjectFromVersionID(versionStr string) (*Project, error) {
```

Comment:
Is this used anywhere?

Reply:
apparently not, but I'm in favor of leaving it because: 
- it's a reasonable operation to want to do.
- the old scheduler did this, and we managed to refactor ourselves away from needing it.
- not having access to projects, or having overly complicated accessors to them, is complicating in a way that I think is worth avoiding.